langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockAnthropicStreamingChatModel.java

```diff
@@ -30,4 +41,36 @@ public enum Types {
             this.value = modelID;
         }
     }
+
+    @Override
+    public void generate(final UserMessage userMessage, final StreamingResponseHandler<AiMessage> handler) {
+        List<ChatMessage> sanitizedMessages = sanitizeMessages(List.of(userMessage));
+        super.generate(sanitizedMessages, handler);
+    }
+
+    @Override
+    public void generate(final List<ChatMessage> messages, final List<ToolSpecification> toolSpecifications, final StreamingResponseHandler<AiMessage> handler) {
+        List<ChatMessage> sanitizedMessages = sanitizeMessages(messages);
+        super.generate(sanitizedMessages, toolSpecifications, handler);
+    }
+
+    @Override
+    public void generate(final List<ChatMessage> messages, final ToolSpecification toolSpecification, final StreamingResponseHandler<AiMessage> handler) {
+        List<ChatMessage> sanitizedMessages = sanitizeMessages(messages);
+        super.generate(sanitizedMessages, toolSpecification, handler);
+    }
+
+    @Override
+    public void generate(final String userMessage, final StreamingResponseHandler<AiMessage> handler) {
```

Comment:
why is this required?

Reply:
I've made some tests with streaming models too, and now I saw that it is not necessary

<img width="528" alt="image" src="https://github.com/user-attachments/assets/dc7a2cb8-0fd2-42ad-ad33-7695d2623089">